### DISCUSSION TITLE
SimpleGrid: How do you fill in the space next to larger grid item

### DISCUSSION THREAD

**Discussion by dennyferra** [chakra-ui/chakra-ui#4587] [Q&A] [answered]
I'm trying to create a responsive grid where items are spread across the grid horizontally (top to bottom) and have one item at least 2x the height of the other grid items. When using SimpleGrid it leaves gaps on the row with the larger item (see attached image). Is there a way to accomplish this so that the items take up the available gaps?

Code:
```jsx
<SimpleGrid minChildWidth="400px" maxW="1280px" gap="4px" margin="0 auto">
  <Box bg="tomato" height="204px">
    Test
  </Box>
  {srces.map((src, i) => (
    <Box bg="tomato" height="100px">
      {i}
    </Box>
  ))}
</SimpleGrid>
```

![grid](https://user-images.githubusercontent.com/244942/129817247-301ec2ac-0023-4bce-919c-b77ca3c677f1.png)

**Comment by dennyferra** [comment, marked as answer]
After playing with this a bit I was able to find my solution:

```jsx
<SimpleGrid
  autoRows="100px"
  gridTemplateColumns="repeat(3, 1fr)"
  maxW="1280px"
  gap="4px"
  margin="0 auto"
>
  <Box bg="tomato" style={{ gridRowStart: 1, gridRowEnd: 3 }}>
    Test
  </Box>
  {srces.map((src, i) => (
    <Box bg="tomato" height="100px">
      {i}
    </Box>
  ))}
</SimpleGrid>
```

Which results in:
![fixed](https://user-images.githubusercontent.com/244942/129923399-92d65730-aad5-466a-b665-5422b71955ba.png)

**Reply by payapula** [reply]
Great that you were able to fix it, in case if you are looking for masonry grid layouts, check these pages for more information - https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Masonry_Layout and https://freefrontend.com/css-masonry-layout-examples/
